alwaysontop/constants.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+    ALWAYSONTOP_DISMISSED: 'dismissed',
```

Comment:
Do we need this event? Why?

Reply:
to know when the window was dismissed

Reply:
How are you consuming this event?

Reply:
i am consuming it from the requiring module to have logic that would prevent the window being displayed if it is dimissed by the user